package.json

```diff
@@ -8,7 +8,8 @@
     "test": "yarn run lint && yarn run jest",
     "jest": "jest",
     "jest:watch": "yarn run jest -- --watch",
-    "prepublish": "babel -d dist/ src/",
+    "build": "babel -d dist/ src/",
+    "prepublish": "npm run build",
```

Comment:
I splitted in order to run build command not only in the publish procedure but also to test locally the library in another project.